src/content/guides/language/evolution.md

```diff
@@ -41,6 +41,33 @@ on the Dart language GitHub repo.
 
 ## Changes in each release
 
+### Dart 3.3
+_Released 15 February 2024_
+| [Dart 3.3 announcement](https://medium.com/dartlang/dart-3-3-325bf2bf6c13)
+
+Dart 3.3 added some enchancements to the language:
+
+* [Extension types][] are a new feature in Dart that allow zero-cost wrapping
+  of existing type. They are similar to wrapper classes and to extension methods,
+  but with implementation difference and benefits over each.
```

Comment:
Would tradeoffs be more accurate?

```suggestion
  but with implementation differences and different tradeoffs.
```

If benefits is more accurate, it seems like "differences" should still be plural here.

```suggestion
  but with implementation differences and benefits over each.
```